freqtrade/commands/arguments.py

```diff
@@ -415,3 +419,10 @@ def _build_subcommands(self) -> None:
                                               parents=[_common_parser])
         webserver_cmd.set_defaults(func=start_webserver)
         self._build_args(optionlist=ARGS_WEBSERVER, parser=webserver_cmd)
+
+        # Add backtesting analysis subcommand
+        analysis_cmd = subparsers.add_parser('backtesting-analysis',
+                                             help='Backtest Analysis module.',
+                                             parents=[_common_parser, _strategy_parser])
```

Comment:
i don't think this command is using any of the strategy methods (`--strategy`, `--strategy-path`, `--recursive-strategy-search`) anymore - so i think we can/should remove _strategy_parser from here, so it's no longer a possible (but then unused) argument.

Reply:
Ah true - I forgot to remove that.